Source/Scene/HereMapsImageryProvider.js

```diff
@@ -0,0 +1,521 @@
+        WebMercatorTilingScheme,
+        defineProperties,
+        DeveloperError,
+        Rectangle,
+        ImageryProvider,
+        defined,
+        CesiumEvent,
+        CesiumMath,
+        Credit,
+        TileProviderError,
+        RuntimeError,
+        Resource,
+        when) {
+    'use strict';
+
+    /**
+     * Provides tiled imagery using the Here Maps Imagery REST API. See https://developer.here.com/documentation/map-tile/topics/request-constructing.html
+     * for more details on how the query URLs are constructed and acceptable values for each option.
+     *
+     * @alias HereMapsImageryProvider.js
+     * @constructor
+     *
+     * @param {Object} options Object with the following properties:
+     * @param {String} [options.baseUrl="aerial.maps.api.here.com"] The URL for accessing tiles, without the Load Balancing prefix.
+     *        You may use the *.maps.cit.api.here.com URLs during development. The first part of the URL determines which tileTypes
+     *.       are available.
+     * @param {String} [options.tileType="maptile"] Determines which type of image will be delivered. E.g. Map, Labels, Streets, etc.
+     * @param {String} [options.scheme="satellite.day"] Specifies the view scheme, e.g normal.day, normal.night, terrain.day, etc.
+              Note, some schemes will be rejected by certain baseUrls.
+     * @param {String} [options.tileFormat="jpg"] Valid values are jpg, png, and png8.
+     * @param {Number} [options.tileSize=256] The width and height of each tile in pixels. Supported values are 256 and 512.
+     * @param {Boolean} [options.forceUseNewest=false] By default, the current map version will be queried only once at start.
+     *        Subsequent tile requests will always use a hash of this version so that tiles are consistent in case the map is updated
+     *        while the app is running. Enabling this flag will cause all tiles to always reqeuest the newest version.
+     *        See https://developer.here.com/documentation/map-tile/topics/tile-version.html#tile-version for details.
+     *
+     *
+     * @see ArcGisMapServerImageryProvider
+     * @see GoogleEarthImageryProvider
+     * @see OpenStreetMapImageryProvider
+     * @see SingleTileImageryProvider
+     * @see TileMapServiceImageryProvider
+     * @see WebMapServiceImageryProvider
+     *
+     * @see {@link https://developer.here.com/documentation/map-tile/topics/overview.html}
+     * @see {@link http://www.w3.org/TR/cors/|Cross-Origin Resource Sharing}
+     *
+     * @example
+     * var here = new Cesium.HereMapsImageryProvider({
+     *     url : 'aerial.maps.cit.api.here.com',
+     *     appId : 'get-yours-at-https://www.developer.here.com/',
+     *     appCode : 'get-yours-at-https://www.developer.here.com/',
+     * });
+     */
+    var HereMapsImageryProvider = function HereMapsImageryProvider(options) {
+        options = defaultValue(options, {});
+
+        //>>includeStart('debug', pragmas.debug);
```

Comment:
`appId` and `appCode` aren't in the doc section as options.

Reply:
Done!